### ISSUE TITLE
Using esri-loader the esri library is loaded on every app view

### ISSUE THREAD

**Issue by markcooney** [Esri/esri-loader#125]
### Expected behavior

- After loading libraries once on the next visit the esri library should be cached
- Fast rendering of esri Map component
- React 15.4.2, esri Javascript 4.8, esri-loader 2.4.0

### Actual behavior

- The esri library is returning a 200 response instead of 304 every time the app is hit
- Was expecting 304 response, esri libary not to load every time
- N/A

### Steps to reproduce the behavior

**We can only help you if we're able to easily reproduce the behavior you describe above.**

Please provide:

 esriLoader.loadModules([
      'esri/Map',
      'esri/widgets/Search',
      'esri/geometry/Point',
      'esri/views/MapView',
      'esri/views/SceneView',
      'esri/geometry/Extent',
      'esri/request',
      'esri/layers/GraphicsLayer',
      'esri/config',
      'esri/core/watchUtils',
      'dojo/_base/array',
    ], options)
     .then(([Map, Search, Point, MapView, SceneView, Extent, esriRequest, GraphicsLayer, esriConfig, watchUtils, array]) => {

Noticed MapView for example returning 200 response every visit to the app and taking 5 secs to load.

Request URL: https://js.arcgis.com/4.8/esri/views/MapView.js
Request Method: GET
Status Code: 200 
Remote Address: 52.85.201.225:443
Referrer Policy: no-referrer-when-downgrade

The esri sandbox 2D example returns a 304 every visit and this renders faster

https://developers.arcgis.com/javascript/latest/sample-code/sandbox/index.html?sample=intro-mapview

Request URL: https://js.arcgis.com/4.8/esri/views/MapView.js
Request Method: GET
Status Code: 304 
Remote Address: 143.204.192.115:443
Referrer Policy: no-referrer-when-downgrade

**Comment by andygup** [comment]
Hi Mark, are you using Chrome? I was able to reproduce this issue in Chrome. It might be a Chrome bug related to the `SPDY` protocol. FWIW, the sandbox is serving up content using HTTP2.

Tested fine on Firefox and Safari.

**Comment by markcooney** [comment]
Thanks Andy , yep using Chrome. I’ll try Firefox tomorrow , in. European
time zone. Great library by the way ,

Thanks Mark

**Comment by tomwayson** [comment]
I can observe this same behavior on the ArcGIS API docs, for example, go to the [Load a basic WebMap](https://developers.arcgis.com/javascript/latest/sample-code/webmap-basic/index.html)  sample, observe a 200 for MapView.js, then click some other sample (like Load a basic WebScene) and then click back to the first sample and you'll see another 200 for MapView.js in Chrome, but 304 in FireFox.

Since this is not an esri-loader issue, I'd like to close this.

@andygup is this tracked anywhere else that @markcooney can be notified of updates?

**Comment by andygup** [comment]
> @andygup is this tracked anywhere else that @markcooney can be notified of updates?

@tomwayson nope, there's no other place to track this since it's possibly a bug in Chrome related SPDY.  

I did some quick searching on chromium.org and didn't find any issues that seemed similar. If @markcooney finds additional info he can still post here even after issue is closed and I'll respond.

Please feel free to close.
